src/test/java/com/epam/tc/hw7/pages/JdiSite.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.tc.hw7.pages;
```

Comment:
Check which classes should be in the test folder and what should be in the main folder. It's not a good idea to put everything in a "test"